sources/packages/backend/libs/utilities/src/_tests_/unit/string-utils.replaceLineBreaks.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { replaceLineBreaks } from "@sims/utilities/string-utils";
+
+describe("StringUtils-replaceLineBreaks", () => {
+  it("Should replace the line break with empty string value in given text when one or more line break character is present.", () => {
+    //Arrange
```

Comment:
Missing white space after the // for lines 5 and 17.

Reply:
Done.